kubeflow/pipelines.py

```diff
@@ -387,13 +393,14 @@ def train_on_synthetic_and_real_dataset(
         "https://storage.googleapis.com/datasetinsights/data/groceries/v3.zip"
     ),
     config: str = "datasetinsights/configs/faster_rcnn_fine_tune.yaml",
+    override: str = "",
     checkpoint_file: str = (
         "https://storage.googleapis.com/datasetinsights/models/Synthetic"
         "/FasterRCNN.estimator"
     ),
     tb_log_dir: str = "gs://<bucket>/runs/yyyymmdd-hhmm",
     checkpoint_dir: str = "gs://<bucket>/checkpoints/yyyymmdd-hhmm",
-    volume_size: str = "100Gi",
+    volume_size: str = "50Gi",
```

Comment:
Why are you reducing memory size?

Reply:
Revert to 100Gi. Don't need to change the volume size.